usaspending_api/download/v2/list_unlinked_awards_downloads.py

```diff
@@ -18,7 +18,7 @@ class ListUnlinkedAwardsDownloadsViewSet(APIView):
 
     redirect_dir = settings.UNLINKED_AWARDS_DOWNLOAD_REDIRECT_DIR
 
-    # This is intentionally not cached so that the latest updates to these files are always returned
+    @cache_response()
```

Comment:
This download is updated as one of the final steps of the nightly pipeline. The cache is cleared after the pipeline finishes, so endpoint will return up to date information as soon as the pipeline finishes. This will prevent us from needing to go to S3 for every request.